docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -18,38 +19,51 @@ The version numbers can be found in these Perl modules:
 Any changes since the last release must be documented in the Changes files.
 Please refer to any Github issues related to the change by the issue number.
 
+ * zonemaster-ldns - [Changes](https://github.com/dotse/zonemaster-ldns/blob/master/Changes)
  * zonemaster-engine - [Changes](https://github.com/dotse/zonemaster-engine/blob/master/Changes)
  * zonemaster-cli - [Changes](https://github.com/dotse/zonemaster-cli/blob/master/Changes)
  * zonemaster-backend - [Changes](https://github.com/dotse/zonemaster-backend/blob/master/CHANGES)
  * zonemaster-gui - [Changes](https://github.com/dotse/zonemaster-gui/blob/master/Changes)
 
 ## 3. Update prerequisites
 
-Make sure the prerequisites section in [README.md](https://github.com/dotse/zonemaster/blob/master/README.md)
-is up to date with regard to [SupportCriteria](https://github.com/dotse/zonemaster/blob/master/docs/internal-documentation/maintenance/SupportCriteria.md).
+Make sure the [declaration of prerequisites](../../../README.md#prerequisites)
+is up to date with regard to [SupportCriteria](SupportCriteria.md).
 
-## 4. Verify that MANIFEST is up to date
+## 4. Update CI configuration
+
+Make sure the Travis configuration for each repo is up to date with the supported Perl versions.
+
+ * zonemaster-ldns - [.travis.yml](https://github.com/dotse/zonemaster-ldns/blob/master/.travis.yml)
+ * zonemaster-engine - [.travis.yml](https://github.com/dotse/zonemaster-engine/blob/master/.travis.yml)
+ * zonemaster-cli - [.travis.yml](https://github.com/dotse/zonemaster-cli/blob/master/.travis.yml)
+ * zonemaster-backend - [.travis.yml](https://github.com/dotse/zonemaster-backend/blob/master/.travis.yml)
+ * zonemaster-gui - [.travis.yml](https://github.com/dotse/zonemaster-gui/blob/master/.travis.yml)
+
+## 5. Verify that MANIFEST is up to date
 
 In order to have a complete installation from a package, the MANIFEST needs
 to be the complete set of files to be included.
```

Comment:
In order to have to create complete, the MANIFEST needs to be the complete set of files to be included. In some cases the MANIFEST file is updated by the `make manifest` target.